e2e/snapshot_ops_test.go

```diff
@@ -0,0 +1,234 @@
+package e2e
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/gluster/glusterd2/e2e/lvmtest"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/restclient"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	snapname = "snaptest"
+)
+
+// TestSnapshot creates a volume and snapshots, runs further tests on it and
+// finally deletes the volume
+func TestSnapshot(t *testing.T) {
+	var err error
+	var brickPaths []string
+	r := require.New(t)
+
+	gds, err = setupCluster("./config/1.toml", "./config/2.toml")
+	if err != nil {
+		r.Nil(err)
+		teardownCluster(gds)
+		return
```

Comment:
don't return in the test cases, let it continue

Reply:
Why do we need to continue in cases where the configuration itself failed. If we continue we may encounter null pointer exceptions and proper cleanup won't happen. I didn't understand why we need to continue here.

Reply:
check other test cases. this is how the test framework as been created

Reply:
@prashanthpai  can you answer @rafikc30  comment, I went through other e2e test cases also, we are not returning an error if setup cluster fails.